webpack/envs/clientNovoDevelopmentConfig.js

```diff
@@ -203,7 +203,13 @@ export const clientNovoDevelopmentConfig = {
     }),
     new HtmlWebpackPlugin({
       filename: path.resolve(basePath, "public", "index.ejs"),
-      template: `!!raw-loader!${path.resolve(basePath, "src", "novo", "src", "index.ejs")}`,
+      template: `!!raw-loader!${path.resolve(
+        basePath,
+        "src",
+        "novo",
+        "src",
+        "index.ejs"
+      )}`,
```

Comment:
(Most of the little changes here are prettier)

Reply:
Prettier is pretty much the worst at formatting, I've been meaning to bring up disabling line length formatting because it rarely makes anything more readable.

Reply:
Def not gonna change that default 80, its everywhere across all of our codebases, we've had previous RFC's on it, etc. Feel free to use inline prettier-disable comments for areas that get funny tho